tools/src/wyvern/target/corewyvernIL/expression/FieldGet.java

```diff
@@ -74,4 +76,9 @@ public Path adapt(View v) {
 			throw new RuntimeException("tried to adapt something that's not a path or type");
 		return new FieldGet((Expression)((Path)objectExpr).adapt(v), fieldName);
 	}
+
+	@Override
+	public Set<String> getFreeVariables() {
+		return new HashSet<>();
```

Comment:
Actually the underlying objectExpr could have free variables.  Thus as a side note IExpr should have getFreeVariables().